opensrp-app/src/main/java/org/smartregister/view/activity/StatsActivity.java

```diff
@@ -57,8 +71,7 @@ protected ViewPager setupViewPager(ViewPager viewPager) {
     public void showProgressDialog(int titleIdentifier) {
         progressDialog = new ProgressDialog(this);
         progressDialog.setCancelable(false);
-        progressDialog.setTitle(titleIdentifier);
```

Comment:
Is the title no longer required? 
If so this can be refactored so that we don't have to pass the `titleIdentifier` in this method